console/console-init/ui/src/modules/address/components/AddressListKebab/AddressListKebab.tsx

```diff
@@ -31,7 +31,7 @@ export const AddressListKebab: React.FunctionComponent<IAddressListKebabProps> =
 }) => {
   const dropdownItems = [
     <DropdownItem
-      id="al-filter-dropdown-item-deleteall"
+      id="addr-list-kebab-deleteall-dropdown"
```

Comment:
We can rename this id as `addr-list-kebab-dropdown-deleteall-item`

Reply:
I think addr-list-deleteall-item-kebab-dropdown will be better?

Reply:
agreed